shared/actions/chat/inbox.js

```diff
@@ -650,7 +655,7 @@ function* _sendNotifications(action: ChatGen.AppendMessagesPayload): Saga.SagaGe
       if (message && message.type === 'Text') {
         console.log('Sending Chat notification')
         const snippet = Constants.makeSnippet(Constants.serverMessageToMessageText(message))
-        yield Saga.put((dispatch: Dispatch) => {
```

Comment:
we use Dispatch globally a lot but its really untyped. The one from redux also doesn't understand thunks